<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Groups</class>
 <widget class="QDialog" name="Groups">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Группы</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#aaaaaa</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView">
         <property name="styleSheet">
          <string notr="true">background-color:#606060</string>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="lineEdit_amount_insert">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;
font-size:13px;</string>
             </property>
             <property name="placeholderText">
              <string>Количество студентов</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_group_insert">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;
font-size:13px;</string>
             </property>
             <property name="placeholderText">
              <string>Название группы</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_special_insert">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;
font-size:13px;</string>
             </property>
             <property name="placeholderText">
              <string>Специальность</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_insert">
             <property name="styleSheet">
              <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;</string>
             </property>
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="comboBox_update">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:#606060; color:white; border-radius:2px</string>
             </property>
             <property name="placeholderText">
              <string>ID поля</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_amount_update">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;
font-size:13px;</string>
             </property>
             <property name="placeholderText">
              <string>Количество студентов</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_group_update">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;
font-size:13px;</string>
             </property>
             <property name="placeholderText">
              <string>Название группы</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_special_update">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;
font-size:13px;</string>
             </property>
             <property name="placeholderText">
              <string>Специальность</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_update">
             <property name="styleSheet">
              <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;</string>
             </property>
             <property name="text">
              <string>Обновить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QComboBox" name="comboBox_delete">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:#606060; color:white; border-radius:2px</string>
           </property>
           <property name="placeholderText">
            <string>ID поля</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>358</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pushButton_delete">
           <property name="styleSheet">
            <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;</string>
           </property>
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
